Samples/Client/WebClient/index.html

```diff
@@ -225,25 +424,25 @@
         }
     }
 
-    var heartbeatInterval = null;
-    function startHeartbeat() {
-        // issue these indefinitely
-        heartbeatInterval = setInterval(function () {
-            try {
-                // note: we don't really care what the response looks like here, so we don't observe it
-                heartbeatGet = new XMLHttpRequest();
-                heartbeatGet.open("GET", server + "/heartbeat?peer_id=" + myId, true);
-                heartbeatGet.setRequestHeader("Peer-Type", 'Client');
-                if (accessToken) {
-                    heartbeatGet.setRequestHeader("Authorization", 'Bearer ' + accessToken);
-                }
-                heartbeatGet.send();
-            } catch (e) {
-                trace("error" + e.description);
-            }
-        }, heartbeatIntervalInMs);
-    }
-
+    var heartbeatInterval = null; 
```

Comment:
mega nit: these changes are kinda interesting. i wonder why your editor touched all these. 😕 ❓

Reply:
inserted tabs instead of spaces